### PR TITLE
fix: plant-group creation

### PR DESCRIPTION
**Pull request by mfloto**
- fixes sensorRanges not being created properly
- fixes return of empty fields

### CODE REVIEW

File: sensor/sensor-model.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ type SensorRange struct {
 }
 
 type SensorRangeChange struct {
-	Sensor string  `json:"sensor"`
+	Sensor string  `json:"sensorType"`
```
Comment (yannickkirschen) [inline sensor/sensor-model.go:24]: Are you sure we want to rename this field? In the database it is SENSOR as well and I thought, a shorter name is better.

Comment (mfloto) [reply sensor/sensor-model.go:24]: Then I must have misread the api.yaml. I will change it back.

**Comment by mfloto** [comment]
This is actually not a full solution to the problem. To make sure we don't get any null values we should check the contents of the request before passing it to the corresponding repository or we do the checks there. Or it could be that the `SENSOR_RANGE` tables' `SENSOR` column does not have the not-null-flag set?

File: go.mod
Comment (mfloto) [inline go.mod:1]: This library needs way too many dependencies...

**Review by yannickkirschen — APPROVED** [review]

**Pull request merged**